hack/ci.Dockerfile

```diff
@@ -0,0 +1,10 @@
+# This Dockerfile WILL used by PROW CI to run E2E tests of the repo
```

Comment:
Duplicate file ?

Reply:
uhum

after merge I will open a CI PR (in openshift/release) to CI to use this new dockerfile; and then, another PR to remove `build/` folder from here, to fix https://github.com/openshift/oadp-operator/issues/1259